History.md

```diff
@@ -101,4 +101,183 @@ MikeGC: WIXBUG:4878 - fix iniutil memory leak
 
 * BMurri: WIXBUG:3902 - Fix ability to find config files in certain circumstances.
 
+<<<<<<< HEAD
```

Comment:
There shouldn't be any modifications to History.md.  Also, there are tons of merge commits in here so it would be nice if you rebased onto develop.